website/content/en/preview/concepts/nodeclasses.md

```diff
@@ -781,8 +813,7 @@ spec:
         snapshotID: snap-0123456789
 ```
 
-The following blockDeviceMapping defaults are used for each `AMIFamily` if no `blockDeviceMapping` overrides are specified in the `EC2NodeClass`
-
```

Comment:
We should still include this line, right?

Reply:
sad an accident